types/cesium/index.d.ts

```diff
@@ -3068,6 +3068,7 @@ declare namespace Cesium {
         depthTestAgainstTerrain: boolean;
         maximumScreenSpaceError: number;
         tileCacheSize: number;
+        tilesLoaded: boolean;
```

Comment:
please provide the details on the change by completing the template. Add the test.